WordPress-Theme/ruleset.xml

```diff
@@ -3,7 +3,7 @@
 	<!-- For more information: https://make.wordpress.org/themes/handbook/review/ -->
 	<description>Standards any Theme to be published on wordpress.org should comply with.</description>
 
-
+	<rule ref="WordPress.Functions.FunctionRestrictions"/>
```

Comment:
For sniffs to be re-usable, it is also important to keep them modular and name them for their specific function.
In this case, may I suggest naming the sniff `DeregisterScript` ?